generators/Generators.fs

```diff
@@ -1289,6 +1289,42 @@ type Series() =
         |> Option.ofNonErrorObject
         |> Option.renderParenthesized
 
+type SgfParsing() = 
+    inherit GeneratorExercise()
+
+    override __.RenderArrange (canonicalDataCase: CanonicalDataCase) =
```

Comment:
You shouldn't have to override this method, converting the expected value should happen in the `RenderExpected` method.